### DISCUSSION TITLE
Change the color of window title bars

### DISCUSSION THREAD

**Discussion by Kodiologist** [lxqt/lxqt#2138] [Q&A] [answered]
Forgive me if this should be obvious, but I can't find a way to customize the color of the window title bars in Lubuntu 21.10. Window title bars appear black on my system, and I'd like them blue. Is there some relevant option in `lxqt-config-appearance`, or is there some way I can customize my Qt theme to change this? I have "Qt Style" set to Breeze and GTK themes set to Adwaita.

**Comment by stefonarch** [comment, marked as answer]
LXQt does not provide a  window manager, which controls window borders, buttons, titles ecc, many different ones can be used:
https://github.com/lxqt/lxqt/wiki/ConfigWindowManagers

Lubuntu ships with openbox, so you can choose a different openbox theme, using `obconf` or `obconf-qt`

**Reply by Kodiologist** [reply]
I see, thanks. I found a theme I like with `obconf`. `obconf-qt` provides largely the same options but doesn't show theme previews, oddly.

**Reply by stefonarch** [reply]
Yes, it's not developed anymore. But actually you can see the effect immediately on other windows.
